### ISSUE TITLE
[Bug] Wrong routes parsing

### ISSUE THREAD

**Issue by ghost** [mkozjak/koa-cache-lite#35]
# Case without 

## Code
```js
const Koa = require('koa');
const cache = require('koa-cache-lite');
const Router = require('koa-router');

const router = new Router({
  sensitive: true,
  strict: true
});

router
  .get('/config', (ctx) => ctx.body = {msg: 'Hello World'})
  .get('/random', (ctx) => ctx.body = {msg: new Date()});

const app = new Koa();

cache.configure(
  {
    '/config': 30000,
// uncomment next line will fix the issue
// '/config/:df': 30000 
  },
  {
    external: {
      type: 'redis',
      host: 'localhost',
      port: 6379,
      keyPrefix: 'cache_'
    },
    debug: true
  }
);

app.use(cache.middleware());
app.use(router.routes(), router.allowedMethods());

app.listen(3000);
```

## Expected result
`localhost:3000/random` should return new date from code

## Actual result
`localhost:3000/random` returns the date from cache.

**Comment by mkozjak** [comment]
@tokenchik thank you for reporting this. It is now fixed in 3.8.6 and will work as expected. `/random` will be cached for 5 seconds by default.

**Comment by ghost** [comment]
@mkozjak, can I avoid caching for endpoints what not in the routes?
I understand your package as cache only for endpoints whats are described at routes. Do I miss something?

**Comment by mkozjak** [comment]
@tokenchik nope, but you can set a route to `false` instead of giving it an integer value, ie. `'/random': false`. PRs to implement such features are also welcome!

**Comment by ghost** [comment]
@mkozjak, thank you so much for fast answers.
Could you recheck my code? If you will uncomment last line in the routes, then `localhost:3000/random` will not be affected by cache.

**Comment by mkozjak** [comment]
@tokenchik tbh, I barely know anymore of the behaviour and the documentation is somewhat lacking. Seems like undefined routes are not cached in the end. If I set something for `/random` it did cache. Can you experiment with those a bit and possibly enhance the documentation? :)

**Comment by ghost** [comment]
@mkozjak 
1) I will be happy to help with your package.
2) Could you reopen this issue?
3) I think full fix for this issue should be published as 4.0.0 version
4) What right behavior should have the package for unlisted route? Cache with default timeout or ignore?
5) The last, not the least I use your package, because i think it cache only listed routes.
